pkgs/applications/office/kmymoney/default.nix

```diff
@@ -1,20 +1,29 @@
 { stdenv, fetchurl, cmake, kdelibs, automoc4, kdepimlibs, gettext, pkgconfig
 , shared_mime_info, perl, boost, gpgme, gmpxx, libalkimia, libofx, libical
-, doxygen }:
+, oxygen_icons, kactivities, aqbanking, perlPackages, gwenhywfar }:
 
 stdenv.mkDerivation rec {
-  name = "kmymoney-4.6.4";
+  name = "kmymoney-4.7.2";
 
   src = fetchurl {
-    url = "mirror://sourceforge/kmymoney2/${name}.tar.xz";
-    sha256 = "04n0lgi2yrx67bgjzbdbcm10pxs7l53srmp240znzw59njnjyll9";
+    url = http://kde.mirrorcatalogs.com/stable/kmymoney/4.7.2/src/kmymoney-4.7.2.tar.xz;
```

Comment:
Nixpkgs knows about the KDE mirror rotation, so you should be able to change this to
```
url = mirrors://kde/stable/kmymoney/4.7.2/src/kmymoney-4.7.2.tar.xz;
```